rfc9114.md

```diff
@@ -1652,21 +1653,21 @@ H3_FRAME_UNEXPECTED.
 
 When a stream cannot be completed successfully, QUIC allows the application to
 abruptly terminate (reset) that stream and communicate a reason; see {{Section
-2.4 of QUIC-TRANSPORT}}. This is referred to as a "stream error." An HTTP/3
+2.4 of QUIC-TRANSPORT}}. This is referred to as a "stream error". An HTTP/3
 implementation can decide to close a QUIC stream and communicate the type of
 error. Wire encodings of error codes are defined in {{http-error-codes}}.
-Stream errors are distinct from HTTP status codes which indicate error
+Stream errors are distinct from HTTP status codes that indicate error
```

Comment:
The RFC editor proposed adding a comma, producing
```suggestion
Stream errors are distinct from HTTP status codes, which indicate error
```
However, that suggests that _all_ HTTP status codes indicate error conditions.  Instead, I've changed which->that indicating that here we're discussing only those HTTP status codes that indicate an error.

Along the way, I've had to learn the difference between the usage of "which" and "that", which still trips me up.

Reply:
I agree the `, which` variant breaks the intention of this sentence.

Reply:
Yep, it's "Stream errors are distinct from (HTTP status codes that indicate...)"